### DISCUSSION TITLE
FNIRSI GC-01: Tube fault alarm

### DISCUSSION THREAD

**Discussion by DimaDoesCode** [Gissio/radpro#176] [Q&A]
Dear all!
Please advise if t's possible to turn off "Tube fault alarm"? I mean "beeps" those occures after 5 mins if nothing was detected. In other words - is continuous mode supported even if nothing detected?
FW: RadPro 2.0.3.
Regards,
Dmitry

**Comment by Gissio** [comment]
Please read the fault section of the user’s manual:

https://github.com/Gissio/radpro/blob/main/docs/manual.md#tube-fault-alarm

**Reply by DimaDoesCode** [reply]
Thank you, but the question actually was about "how to turn it off" if possible of course.
Regards,
Dmitry
P.S. And happy New Year!

**Reply by Gissio** [reply]
DimaDoesCode: it my understanding that you are a native Russian speaker. Could you check the Russian translation in 2.2beta3?

The strings are available here: https://github.com/Gissio/radpro/blob/main/platform.io/src/strings/ru.h

If you are willing to help us with these, please follow the discussion here: https://github.com/Gissio/radpro/discussions/241
